notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        core = ov.Core()
</pre><p>we have risk to reference core before assignment below as it created only if model converted in previous run</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>yes you are right, I will move it out of  if else loop.</p>